README.md

```diff
@@ -31,51 +28,103 @@ Imagine a declarative Kubernetes-style API for _anything_, supported by an ecosy
 
-Our `kcp` specific patches are in the [feature-logical-clusters-1.22](https://github.com/kcp-dev/kubernetes/tree/feature-logical-clusters-1.22) feature branch in the [kcp-dev/kubernetes](https://github.com/kcp-dev/kubernetes) repo. See [DEVELOPMENT.md](DEVELOPMENT.md) for how the patches are structured and how they must be formatted during our experimentation phase.  See [GOALS.md](GOALS.md) for more info on how we intend to use `kcp` as a test-bed for exploring ideas that improve the entire ecosystem.
+## Can `kcp` do anything else?
 
+Yes! Here are a few of our top-level goals:
 
-## What's in this repo?
+### Host thousands (*) of small-ish Kubernetes-like [logical clusters](docs/investigations/logical-clusters.md) in a single instance
 
-First off, this is a prototype, not a project. We're exploring these ideas here to try them out, experiment, and bounce them off each other.  Our [basic demo](contrib/demo/README.md) leverages the following components to show off these ideas:
+* Orders of magnitude smaller than a typical large, shared, multi-tenant cluster (~50 resources per cluster)
+* Inexpensive - nearly for free in terms of resource utilization & cost for an empty cluster
+* Independent - install different versions of a CRD in each logical cluster (!!!)
+* Per-cluster administrative rights - each "owner" (person/team/group) of a cluster is a full admin
 
-- **`kcp`**, which serves a Kubernetes-style API with a minimum of built-in types.
-- **`cluster-controller`**, which along with the `Cluster` CRD allows `kcp` to connect to other full-featured Kubernetes clusters, and includes these components:
-  - **`syncer`**, which runs on Kubernetes clusters registered with the `cluster-controller`, and watches `kcp` for resources assigned to the cluster
-  - **`deployment-splitter`**, which demonstrates a controller that can split a `Deployment` object into multiple "virtual Deployment" objects across multiple clusters.
-  - **`crd-puller`** which demonstrates mirroring CRDs from a cluster back to `kcp`
-- **`virtual-workspaces`** which demonstrates how to implement apiservers for custom access-patterns, e.g. like a workspace index.
-- **`syncer`** which implmenents a transparent multi-cluster sync of resources to offer compute (i.e. to run pods and other workload types) to kcp workspaces using external Kubernetes clusters.
-- **`kubectl-kcp`** which is a kubectl plugin that offers kcp specific sub-commands.
+### Treat compute as a utility
+* [Transparent multi-cluster](docs/investigations/transparent-multi-cluster.md) - run `kcp` as a control plane in front of your physical compute clusters for workloads, and let `kcp` determine how to schedule your workloads to physical compute clusters
+* Dynamically add more **compute** capacity as demand increases - not just nodes, but entire Kubernetes clusters
 
-## So what's this for?
+### Massive scale
```

Comment:
"massive scale" sounds interesting and challenging.  A few more words in this README on how it is being addressed would be welcome.